DataFormats/SiStripCluster/interface/SiStripApproximateCluster.h

```diff
@@ -26,20 +27,26 @@ class SiStripApproximateCluster {
                                      float hitPredPos,
                                      bool peakFilter);
 
-  cms_uint16_t barycenter() const { return barycenter_; }
+  float barycenter() const {
+    float _barycenter = compBarycenter_ * maxBarycenter_ / maxRange_;
```

Comment:
Please avoid underscore as the first character (2.14 in https://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1)